.travis.yml

```diff
@@ -39,6 +39,20 @@ jobs:
         sed -i 's/\\n$//g' "$TESTS_FILES_PREFIX"*
         popd
         ./gradlew scalaStyle testScoverageClasses --parallel
+    - stage: 'compile helloworld project'
+      name: 'compile helloworld project'
+      script: |
+        find **/helloworld/src/test/scala/* -type f -name '*.scala' | cut -d "/" -f5-$2 | sed 's/.\{6\}$/\*/g' > $TESTS_FILE_NAME
```

Comment:
I dont think any of that complexity for helloworld. Simply do `cd helloworld && ./gradlew scalaStyle testScoverageClasses --parallel"

Reply:
fixed